### ISSUE TITLE
Denial of Service sercurity vulnerabilities

### ISSUE THREAD

**Issue by fongzihao1992** [kdeloach/react-lineto#44]
Path: react-lineto > prop-types > fbjs > isomorphic-fetch >-fetch

**Comment by stahlmanDesign** [comment]
```node-fetch  <=2.6.0 || 3.0.0-beta.1 - 3.0.0-beta.8
Denial of Service - https://npmjs.com/advisories/1556
fix available via `npm audit fix --force`
Will install react-lineto@3.0.0, which is a breaking change
node_modules/node-fetch
  fetch-ponyfill  1.0.0 - 6.0.2
  Depends on vulnerable versions of node-fetch
  node_modules/fetch-ponyfill
    embed-video  >=2.0.0
    Depends on vulnerable versions of fetch-ponyfill
    node_modules/embed-video
  isomorphic-fetch  2.0.0 - 2.2.1
  Depends on vulnerable versions of node-fetch
  node_modules/isomorphic-fetch
    fbjs  0.7.0 - 1.0.0
    Depends on vulnerable versions of isomorphic-fetch
    node_modules/fbjs
      prop-types  15.5.0-alpha.0 - 15.6.1
      Depends on vulnerable versions of fbjs
      node_modules/react-lineto/node_modules/prop-types
        react-lineto  *
        Depends on vulnerable versions of prop-types
        Depends on vulnerable versions of react
        node_modules/react-lineto
```

**Comment by kdeloach** [comment]
I'm far removed from frontend development these days. Mind explaining how to fix this? Thanks.

**Comment by stahlmanDesign** [comment]
Not sure. I can't see where react-lineto uses node-fetch, so I think the problem is not with you lib, but with nested dependencies. It might be as simple as updating prop-types and react, to more recent/latest versions

**Comment by bkw** [comment]
These should be resolved in #50 .
